apps/src/templates/manageStudents/ManageStudentsLoginInfo.jsx

```diff
@@ -0,0 +1,75 @@
+import React, {Component, PropTypes} from 'react';
+import LoginTypeParagraph from '@cdo/apps/templates/teacherDashboard/LoginTypeParagraph';
+import {SectionLoginType} from '@cdo/apps/util/sharedConstants';
+import i18n from "@cdo/locale";
+
+const styles = {
+  explination: {
+    clear: 'both',
+    paddingTop: 20,
+  }
+};
+
+class ManageStudentsLoginInfo extends Component {
+  static propTypes = {
+    sectionId: PropTypes.number,
+    sectionCode: PropTypes.string,
+    loginType: PropTypes.string,
+  };
+
+  render() {
+    const {loginType, sectionId, sectionCode} = this.props;
+    return (
+      <div style={styles.explination}>
+        <h2>Login information</h2>
+        {(loginType === SectionLoginType.word || loginType === SectionLoginType.picture) &&
+          <div>
+            <p>
+              {i18n.sectionCode() + ': ' + sectionCode}
```

Comment:
Is there a reason i18n string sometimes include the : and sometimes you add them in line?

Reply:
Mainly because I wanted to use this string that already existed.